web/src/pages/dashboard/AdapterPage/EditAdapter.js

```diff
@@ -1,157 +1,148 @@
-                                required: true,
-                                message: 'The id is required!',
-                            },
-                        ]}
-                        initialValue={this.props.location.state.id}
-                    >
-                        <Input />
-                    </Form.Item>
-                    <Form.Item
-                        name={['adapter', 'name']}
-                        label="Name"
-                        rules={[
-                            {
-                                required: true,
-                                message: 'The name is required!',
-                            }]
-                        }
-                        initialValue={this.props.location.state.name}
-                    >
-                        <Input />
-                    </Form.Item>
-                    <Form.Item
-                        name={['adapter', 'type']}
-                        label="Type"
-                        rules={[
-                            {
-                                required: true,
-                                message: 'The type is required!',
-                            }]
-                        }
-                        initialValue={this.props.location.state.type}
-                    >
-                        <Input />
-                    </Form.Item>
-                    <Form.Item
-                        name={['adapter', 'param1']}
-                        label="Parameter 1"
-                        rules={[
-                            {
-                                required: true,
-                                message: 'The Parameter 1 is required!',
-                            },
-                        ]}
-                        initialValue={this.props.location.state.param1}
-                    >
-                        <Input />
-                    </Form.Item>
-                    <Form.Item
-                        name={['adapter', 'param2']}
-                        label="Parameter 2"
-                        rules={[
-                            {
-                                required: true,
-                                message: 'The Parameter 2 is required!',
-                            },
-                        ]}
-                        initialValue={this.props.location.state.param2}
-                    >
-                        <Input />
-                    </Form.Item>
-                    <Form.Item {...tailFormItemLayout}>
-                        <Button type="primary" htmlType="submit">
-                            Submit
-                        </Button>
-                    </Form.Item>
-                </Form>
-            </Card>
-        )
-    }
+    const tailFormItemLayout = {
+      wrapperCol: {
+        xs: {
+          span: 24,
+          offset: 0,
+        },
+        sm: {
+          span: 16,
+          offset: 8,
+        },
+      },
+    };
```

Comment:
I recommend to use `{...tailFormItemLayout}` to fix the Button layout of the FormItem.
But if you decide to use your method, please delete these lines.

Reply:
I think this method will help in realigning the button easily, if we want to change the style next time
I will do the required changes

Reply:
@Xiaoyang-Gu